src/tools/cephfs/top/cephfs-top

```diff
@@ -413,7 +413,9 @@ class FSTop(FSTopBase):
                 endmenu = True
             elif key == ord('q'):
                 self.stdscr.erase()
-                if self.active_screen == FS_TOP_ALL_FS_APP:
+                if not fs_list:  # no fs, navigate to home screen
```

Comment:
I think it will be good if we can do the same thing for `choose_field()` too?

Reply:
This seems to be a bigger issue. So created tracker [1] to fix it separately.

[1] https://tracker.ceph.com/issues/58813